tasks.py

```diff
@@ -86,7 +86,7 @@ def _login_to_cf(ctx, space):
 
 def _do_deploy(ctx, space):
 
-    manifest_filename = f"manifest-{space}.yml"
+    manifest_filename = f"manifests/manifest-{space}.yml"
```

Comment:
I think the new directory didn't get added

Reply:
Thanks @toddlees! Fixed